### PR TITLE
mod: force ipv4 and allow insecure-registries

### PR DESCRIPTION
**Pull request by yaroslavya**

### CODE REVIEW

**Comment by yaroslavya** [comment]
@fabianhelm need a review

File: daemon.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "ipv6": false,
```
Comment (fabianhelm) [inline daemon.json]: lets keep ipv6 enabled

**Review by fabianhelm — COMMENTED** [review]
please also set a fixed docker version to the previous stable version??

File: daemon.json (hunk 1 above)
Comment (yaroslavya) [reply daemon.json]: @fabianhelm done!

**Comment by yaroslavya** [comment]
@fabianhelm all done: 
- ipv6 set to true explicitly
- docker image changed from latest to explicit. The [latest](https://docs.docker.com/engine/release-notes/24.0/#2405) (24.0.5) has really introduced a docker-compose version change. I've adjusted the readme to reflect these changes and so we remember to revert them once its fixed.

need a review again :memo:

**Review by fabianhelm — APPROVED** [review]
looks good, thanks

**Pull request merged**